model/src/test/kotlin/config/PackageConfigurationTest.kt

```diff
@@ -180,5 +220,66 @@ class PackageConfigurationTest : WordSpec({
                 REPOSITORY_PROVENANCE
             ) shouldBe true
         }
+
+        "return true if the package configuration contains only the identifier and the version is in range" {
+            val config =
+                PackageConfiguration(
+                    id = Identifier.EMPTY.copy(name = "some-name", version = "[51.0.0,60.0.0]")
+                )
+
+            config.matches(
+                config.id.copy(version = "55"),
+                ARTIFACT_PROVENANCE
+            ) shouldBe true
+        }
+
+        "return true if the package configuration contains only the identifier and the version is not in range" {
+            val config =
+                PackageConfiguration(
+                    id = Identifier.EMPTY.copy(name = "some-name", version = "[51.0.0,60.0.0]")
+                )
+
+            config.matches(
+                config.id.copy(version = "6"),
+                ARTIFACT_PROVENANCE
+            ) shouldBe false
+        }
+
+        "return false if the version is in range and the vcs source origin doesn't match" {
```

Comment:
I think this test case should simply use an identifier without a version range and be named:

`return false if only the source code origin is not equal`

I believe we missed to add that test when `sourceCodeOrigin` has been added.
Ideally this test case was added in a separate commit, preceeding this commit.